engine/src/main/resources/assets/schemas/schemaPlayerSpawnConfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Settings for different ships, classifying the hull and items/money given to the player on spawn.",
+  "type": "object",
+  "properties": {
+    "hull": {
+      "type": "string",
+      "description": "Id of the hull of this ship.",
+      "pattern": "^\\w+:\\w+$"
+    },
+    "items": {
+      "type": "string",
+      "description": "Space separated list of items the player possesses on spawn. See /docs/items.txt for more detailed explanation."
```

Comment:
From looking in other spawn configs I'd say this field needs a regex for it.

Reply:
It would probably be better to change how the items are defined here yeah, for now though this regex works